pkg/server/commands/reverseexpand/reverse_expand_test.go

```diff
@@ -141,7 +141,75 @@ type document
 		} else {
 			require.Nil(t, res)
 		}
+		<-done
```

Comment:
can you add a comment here that explains why we need this? this seems to be waiting for the goroutine above to avoid a leak, but you didn't add `defer goleak.VerifyNone(t)` so i'm not sure